pkg/reconciler/service/config/testdata/config-network.yaml

```diff
@@ -0,0 +1,136 @@
+# Copyright 2018 The Knative Authors
```

Comment:
There are generally relative symlinks not copies.

This is also something that should be in the follow-up PR.